elide-async/src/main/java/com/yahoo/elide/async/service/AsyncQueryCleaner.java

```diff
@@ -0,0 +1,38 @@
+/*
+ * Copyright 2020, Yahoo Inc.
+ * Licensed under the Apache License, Version 2.0
+ * See LICENSE file in project root for terms.
+ */
+package com.yahoo.elide.async.service;
+
+import java.util.concurrent.Executors;
+import java.util.concurrent.ScheduledExecutorService;
+
+import javax.inject.Singleton;
+
+/**
+ * Class for initializing the Query Cleaner.
+ */
+@Singleton
+class AsyncQueryCleaner {
```

Comment:
Do we need both a a Cleaner and a CleanerService?  Why not just combine these into a single concept?